com.unity.cinemachine/Samples~/3D Samples/ThirdPersonWithRoadieRun.unity

```diff
@@ -2512,14 +2512,26 @@ PrefabInstance:
       propertyPath: m_ControllerManager.Controllers.Array.data[3].Input.LegacyInput
       value: Fire3
       objectReference: {fileID: 0}
+    - target: {fileID: 7804758404662293724, guid: 4d8d7a9a98d3f4ac2967d48094ea010f, type: 3}
+      propertyPath: m_ControllerManager.Controllers.Array.data[4].Input.InputAction
+      value: 
+      objectReference: {fileID: -1608118824973676762, guid: 1d6e640e716dc4ff6989b73d02023f2b, type: 3}
```

Comment:
Why does it reference InputAction?

Reply:
because the action mappings are serialized, so they get reloaded when you reopen the scene